### ISSUE TITLE
The Navigation>Settings screen should have different sound settings

### ISSUE THREAD

**Issue by raindropsfromsky** [osmandapp/Osmand#6296]
If we press the **gear wheel** icon in the **Set a new route** dialog, we see the following screen:
![screenshot_20181122-143631](https://user-images.githubusercontent.com/9047168/48892929-2da9a400-ee65-11e8-8494-279f5a73f0ec.jpg)

Let us call this "Route settings" screen. 
The purpose of this screen to set up the initial settings for the whole trip. 

The **audio** section of this screen has three issues:

**1.  Move the sound on/off button to the map screen**</br> </br>While driving, the user would need to mute the sound at a moment's notice. This is NOT a fixed choice for the whole trip. The driver should be able to mute the sound immediately. Therefore, the **sound on/off** toggle button is not needed in the **Route settings** screen. </br> </br>Instead, please provide a **speaker/mute** button on the map screen.  During navigation, it should be hidden, but when the user taps the map screen, it should pop up (just like the **map orientation** control). Thus the driver has to tap the  device twice to mute the speaker (He taps anywhere on the map screen to make the button visible, and then taps on it to mute the sound.) </br> </br>I know this can be done through **Quick Actions**, but I feel it should be a standard setting.

**2.  Move the voice guidance language control to the Navigation settings screen**</br> </br>The user would set his voice guidance language only once after he installs OsmAnd. He would not change it on a trip-by-trip basis.  </br> </br>Therefore the language-selection control should be removed from this screen. (It belongs to the **Navigation controls** screen)

**3. Move the pause music option to the Navigation settings screen** </br> </br>The third control is to choose whether to pause music. Even this is a one-time selection, not a "per-trip" choice. Therefore this control also should be removed from this screen. Let it be in the **Navigation controls** screen.

*********
**4. Add  "frequency of voice guidance" control to the Route settings screen**</br> </br>On the other hand, the "Route settings" screen does not provide a much-needed control: The frequency of voice guidance.

If the route is familiar, the user typically does not need voice guidance at all. 
But the more unfamiliar the route, the higher the frequency of voice guidance he needs. 

In other words, the driver needs to set the frequency of voice guidance on a "per trip" basis. Therefore this control should be provided on the "Route settings" screen.

**Comment by sonora** [comment]
Related: #4917, because 'Voice guidance output' is a more transient setting depending on the current situation when you start the navigation, and deserves to be on that screen.

**Comment by raindropsfromsky** [comment]
In general, the logic should be-
1. If the setting is common for all trips, move it to user preferences>Navigation settings.
2. If the setting is fixed for the whole trip, move the control to "Trip(route) settings" screen
3. If the user has to change the setting during the trip, provide it on the map screen.

**Comment by dmpr0** [comment]
I think we can close this issue because we made Navigation UI redesign and all Settings screens too.
